src/core/config/srp_server.h

```diff
@@ -45,6 +45,16 @@
 #define OPENTHREAD_CONFIG_SRP_SERVER_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_SRP_SERVER_UDP_PORT
+ *
+ * Specifies the SRP Server UDP port.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_SRP_SERVER_UDP_PORT
+#define OPENTHREAD_CONFIG_SRP_SERVER_UDP_PORT 53535
```

Comment:
Dynamic UDP ports range from 49152 to 65535. If this port is larger than 49152, we need to skip 53535 in `Udp::GetEphemeralPort`. 
That being said, I would suggest to use a UDP port < 49152.

Thoughts?

Reply:
oh, that is a mistake, I think we should use `35353`.

Reply:
The Border Agent port (`49191`) and Backbone Router port (`61631`) are allocated in this range as well.